### PR TITLE
LPS-168954 HR_511 SR shouldn't read "No Conditions yet" as "heading level 1" under add user segment page

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
The sizes and types of the rendered message "No conditions yet" in the Segments Editor component is incorrect from and Accessibility perspective.
[Link to the story](https://issues.liferay.com/browse/LPS-168954)

Proposed Solution
========
The proposed solution consists in using the ClayEmptyState component from clay which already has the correct format to render the message properly. Also adds value by using a standardized component instead of a custom component.

Steps to Verify:
----------------
1. Go to Product Menu -> People -> Segments
2. Click on '+' button to create a new segment
3. Check that the EmptyState is correctly displayed
4. Try to add different properties to the segment and remove them to see if it is correctly rendered

Screenshots
----------------
Screenshot 2023-01-11 at 14 29 49

*No tests need to be added since we're using a clay component and its already tested.

### CODE REVIEW

**Comment by Stephy6** [comment]
ci:test:sf

**Review by claraizquierdo — COMMENTED** [review]
Hi @Stephy6 ,

Great work and super fast! I just left a small comment but everything is working fine.

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/EmptyPlaceholder.es.js [hunk 1]
```diff
@@ -12,24 +12,21 @@
...
  */
 
+import ClayEmptyState from '@clayui/empty-state';
 import React from 'react';
 
 import {sub} from '../../utils/utils.es';
 
 export default function EmptyPlaceholder() {
 	return (
 		<div className="empty-contributors mb-0 p-5 rounded taglib-empty-result-message">
```
[the remark this answers is not in the document]
Comment (Stephy6) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/EmptyPlaceholder.es.js:22]: Done, thank you

**Comment by Stephy6** [comment]
ci:test:relevant

**Review by Mikellorza — APPROVED** [review]
The solution is working fine for me. Now I can see the text with less size.

**Review by claraizquierdo — APPROVED** [review]
Great work 🎉

**Comment by Stephy6** [comment]
ci:test:relevant

**Comment by Stephy6** [comment]
ci:forward

**Pull request closed without merging**